lib/matplotlib/ticker.py

```diff
@@ -1238,40 +1258,39 @@ def format_eng(self, num):
         u'-1.00 \N{GREEK SMALL LETTER MU}'
 
         `num` may be a numeric value or a string that can be converted
-        to a numeric value with the `decimal.Decimal` constructor.
+        to a numeric value with `float(num)`.
```

Comment:
should use double backquotes (single backquotes are for things that refer to other entries in the docs).